CONTRIBUTING.md

```diff
@@ -5,6 +5,12 @@ developers and one time contributors.  This guide should help you to create
 contributions that can be easily integrated into the FreeOrion project.
 
 
+## Respect the FreeOrion Vision Statement when contributing
+
+All contributions should follow the spirit of the [FreeOrion Vision Statement].
+We can't consider any contributions that miss or oppose the Vision Statement.
```

Comment:
don't like this. if nothing else, its meaning is vague to the point of uselessness / meaninglessness